README.md

```diff
@@ -22,6 +22,8 @@ The public outrage that followed Ang's "light" sentencing gave San Francisco Dis
 
 Two weeks and three days after the news broke over Randy's sentencing, with the anger of how "weak" it was fresh in their minds, people first heard of my accident in the news, igniting a furor of angry, hateful comments directed at me and at cyclists in general.
 
+Through a series of leaks and press releases, George Gascón, San Francisco's District Attorney, built a damning strawman case against me in the local media. In response to the public sentiment he curated, he then charged me with felony vehicular manslaughter, the first time in history that this charge was applied to someone for riding a bicycle. 
```

Comment:
… the first time in history that this charge [change "was" to "had been"] applied to someone [delete "for"] riding a bike